src/fdb5/io/LustreSettings.cc

```diff
@@ -35,19 +38,24 @@ bool fdb5LustreapiSupported() {
 #endif
 }
 
-int fdb5LustreapiFileCreate(const char* path, LustreStripe stripe) {
+int fdb5LustreapiFileCreate(eckit::PathName path, LustreStripe stripe) {
 
 #if defined(fdb5_HAVE_LUSTRE)
 
-    static bool lustreapi_silence = false;
+    struct statfs buf;
 
-    if(not lustreapi_silence) {
-        fdb5_lustreapi_silence_msg();
-        lustreapi_silence = true;
-    }
+    statfs(path.dirName().localPath(), &buf);
```

Comment:
Is this where we want to do this (i.e. on every file creation), or do we want to do it at the root/directory level? I don't have strong opinions.